implementations/rust/ockam/ockam_entity/src/lib.rs

```diff
@@ -0,0 +1,340 @@
+use crate::EntityError;
```

Comment:
Please add the lib header like other crates that has all of the deny attributes etc. Place holder text for missing documentation is fine, we can iterate on it here.